checklists/symfony-release-checklist.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Symfony Release Checklist
+category: Back-End
+date: '2020-03-11'
+tags: ['symfony', 'release']
+description: The list of the To-do's we have complete right before the deploy our app to stage or live 🚀
+---
+
+- [ ] Check dependencies in the composer.json file
+      Make sure the development and production dependencies are correctly defined and production release does not include development packages
+
+- [ ] Check the environment values
+      Check environment values in the `.env` file to prevent a leak of any sensitive information make sure there are no real data defined
```

Comment:
```suggestion
Check environment values in the `.env` file to prevent a leak of any sensitive information make sure there are no real data defined
```